.gitlab/build/docker_install.sh

```diff
@@ -5,9 +5,12 @@
 set -xe
 
 apt-get update -yqq
-apt-get install git libzip-dev unzip libxml2-utils wget -yqq
+apt-get install git libzip-dev unzip parallel libxml2-utils wget -yqq
 
 php -r "readfile('http://getcomposer.org/installer');" | php -- --install-dir=/usr/local/bin/ --filename=composer
 chmod +x /usr/local/bin/composer
 
-docker-php-ext-install zip
+curl --location --output /usr/local/bin/phpunit https://phar.phpunit.de/phpunit.phar
+chmod +x /usr/local/bin/phpunit
```

Comment:
Why are you installing an additional PHPUnit instance? Why not use the one in `vendor/bin/phpunit`?